src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReferralHelperTest.java

```diff
@@ -587,9 +592,13 @@ private ReferralReplyTypeItem createReferralReplyTypeItem(String id) {
         referralReplyType.setReplyToEmailAddress("replyToEmail");
         referralReplyType.setDirectionTo("directionTo");
         referralReplyType.setReplyDocument(List.of(createDocumentType("1")));
+        referralReplyType.setReplyDateTime("replyDateTime");
```

Comment:
can this be an actual date time rather than dummy text

Reply:
Done